packages/showcases/vue/stories/components/selection-controls/VtmnQuantity/VtmnQuantity.stories.js

```diff
@@ -0,0 +1,76 @@
+import { VtmnQuantity } from '@vtmn/vue';
+import { parameters } from '@vtmn/showcase-core/csf/components/selection-controls/quantity.csf';
+
+export default {
+  title: 'Components / Selection controls / VtmnQuantity',
+  component: VtmnQuantity,
+  argTypes: {
+    identifier: {
+      type: { name: 'string', required: true },
+      description: 'Id of the input.',
+      defaultValue: 'quantity-story',
+      control: {
+        type: 'text',
+      },
+    },
+    label: {
```

Comment:
Why don't you use argTypes from the quantity.csf.js file?

Reply:
Because I don't use the `value` property.